README.md

```diff
@@ -64,27 +64,27 @@ Additionally, while CLIP1 and 2 both gave the top one classification prediction
 - the data from this run also provides the second most probable category the model assigned to each entry, which can shed light on the model's behaviour with this type of a dataset
 - the model used in the third run was already customised based on human-annotated feedback on the previous two
 
 ### Primary data
 A spreadsheet containing (per image):
 - its URL (from a database containing all images from the dataset; for qualitative close-reading analysis)
 - CLIP3 first-choice category and the certainty score of that classification (score is based on training data; for CLIP3, this is human-produced image classification annotations)
-- CLIP3 second-choice category and the certainty score of that classification (-,,-)
+- CLIP3 second-choice category and the certainty score of that classification
 
 ### Secondary data
-(used for refining the RQ and hypotheses; would recommend for exploration of H2, H3)
 A spreadsheet containing (per every image from CLIP3 first-choice):
 - its URL (see Primary data)
 - CLIP3 first-choice category (see Primary data)
-- evaluation score of that classification (i.e., a percentage score expressing the probability of a human annotator classifying an image as belonging in the same first-choice category as CLIP3 did; based on human-annotated control batch produced by the group)<br>
-Various visualisations based on the above spreadsheet (created during the hackathon)
+- evaluation score of that classification (i.e., a percentage score expressing the probability of a human annotator classifying an image as belonging in the same first-choice category as CLIP3 did; based on human-annotated control batch produced by the group)
+
+Various visualisations based on the above spreadsheet (created during the Hackathon)
 
 
 ## Research question(s), hypotheses
-RQ: Do the certainty scores in CLIP3 predictions correspond to the actual categories and/or visual form of the images classified, and, if so, then what are the tendencies? Is there significant variation in performance between clategories? Are there CLIP3 predictions not corresponding to human classification in cases classified by CLIP with low/high confidence? What can we conclude about the performance of a model thus customised?
+RQ: Do the certainty scores in CLIP3 predictions correspond to the actual categories and/or visual form of the images classified, and, if so, then what are the tendencies? Is there significant variation in performance between categories? Are there CLIP3 predictions not corresponding to human classification in cases classified by CLIP with low/high confidence? What can we conclude about the performance of a model thus customised?
 
 **H0:** There are no significant patterns in the performance of CLIP3 across certainty levels and categories.
```

Comment:
does *fals.* really need to be shortened?

Reply:
I'd say _falsification_ -> keep